src/components/career-form/career-form.tsx

```diff
@@ -110,119 +139,162 @@ export const CareerForm: React.FC<CareerFormProps> = (props) => {
+              Lade hier bitte deine relevanten Dokumente hoch, wie zb Lebenslauf
+              (CV), Motivationschreiben oder Referenzen. Erlaubt sind
+              ausschließlich 3 PDF Dateien, die maximale Größe pro Datei beträgt
+              20MB.
+            </InfoTextContainer>
           </GridItem>
 
+          {/*Privacy-Policy Checkbox*/}
           <GridItem>
-            <CaptionText>
-              <Sup>*</Sup> Pflichtfeld
-            </CaptionText>
+            <Container>
+              <CheckboxContainer>
+                <Checkbox
+                  type="checkbox"
+                  id={'privacy-policy'}
+                  {...register('privacy', {
+                    required: 'Deine Zustimmung fehlt',
+                  })}
+                />
+                <CheckboxLabel htmlFor="privacy-policy">
+                  {privacyChecked && <CheckboxMark />}
+                  <PolicyText>
+                    Hiermit bestätige ich, dass ich die{' '}
+                    <TextLink to={PRIVACY_POLICY}>
+                      Datenschutzerklärung
+                    </TextLink>{' '}
+                    zur Kenntnis genommen habe. <Sup aria-hidden={true}>*</Sup>
+                  </PolicyText>
+                </CheckboxLabel>
+              </CheckboxContainer>
+              {errors.privacy && <FormError error={errors.privacy} />}
+            </Container>
           </GridItem>
 
           <GridItem>
-            <ProgressBar
-              isSubmitting={isSubmitting}
-              progress={uploadProgress}
-            />
             <Actions
               tryAgainFn={tryAgain}
               isSubmitting={isSubmitting}
               error={errors.api}
+              fieldErrors={errors}
             />
           </GridItem>
+
+          <GridItem>
+            <FileContainer>
+              <CaptionText>
+                <Sup>*</Sup> Pflichtfeld
+              </CaptionText>
+            </FileContainer>
+          </GridItem>
         </Grid>
       </Fieldset>
     </CareerFormStyled>
   );
 };
+
+const Container = styled.div`
+  margin-bottom: 24px;
+`;
+
+export const Headline = styled(TextTitle)`
+  margin-top: 40px;
+  margin-bottom: 29px;
+
+  ${up('md')} {
+    margin-top: 80px;
+  }
+`;
+
+const InfoTextContainer = styled.div`
+  display: flex;
+  align-items: left;
+  flex-direction: row;
+  margin: 24px 0px;
+  line-height: 24px;
+`;
+
+const PolicyText = styled.div`
+  font-size: 14px;
+  line-height: 110%;
+  a {
+    font-size: 14px;
+    line-height: 110%;
+  }
+`;
+
+const CheckboxLabel = styled.label`
+  display: flex;
+  position: relative;
+  flex-direction: row;
+  cursor: pointer;
+
+  div {
+    width: calc(100% - 40px);
+  }
+
+  &:before {
+    content: '';
+    display: inline-block;
+    position: relative;
+    top: -3px;
+    width: 20px;
+    height: 20px;
+    margin-right: 20px;
+    background: rgba(122, 143, 204, 0.3);
+    border-radius: 4px;
+  }
+
+  &:hover {
+    &:before {
```

Comment:
**First**
pseudo classes and pseudo elements have actually different amount of colons. 
+ pseudo classes like hover: `:hover`
+ pseudo elements like before: `::before`

Here some links:
+ https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-elements
+ https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-classes

Why does it still work here? Because of this:
> most browsers support both syntaxes for the original pseudo-elements.

This compatibility mesure creates the confusion around the colons I guess (and I share that confusion too)

**Second**
Do yourself a favor and keep nesting to a minimum. It helps the readability and you can perfectly write `&:hover::before {}` here